src/curve/stable/StablePair.sol

```diff
@@ -150,7 +152,7 @@ contract StablePair is ReservoirPair {
     function burn(address to) public nonReentrant returns (uint256 amount0, uint256 amount1) {
         _syncManaged();
 
-        (uint256 balance0, uint256 balance1) = _balance();
+        (uint256 _reserve0, uint256 _reserve1, ) = _getReserves();
```

Comment:
Unrelated but StablePair has _getReserves() and getReserves(). Should we delete the private one and just use the public one?